src/test/mina_automation/README.md

```diff
@@ -0,0 +1,21 @@
+mina automation
+==============
+
+The `mina_automation` lib is a small utility which helps to automate local app execution
+
+Library can be used in various local test and automation. For example to test if archive and extract blocks apps gives expected archive state. Main feature of an lib is ability to detect location of apps. App can be build locally and resides in _build/default folder, in /usr/local/bin folder.
+
+It also has automation for docker if we would like to run docker command locally and use some output further in the test.
+
+An example of usage:
```

Comment:
Could you add description of what this example performs exactly?

Also, what are `~blocks` and `~archive_uri`?